src/afl-fuzz-queue.c

```diff
@@ -268,14 +270,17 @@ void update_bitmap_score(afl_state_t *afl, struct queue_entry *q) {
 void cull_queue(afl_state_t *afl) {
 
   struct queue_entry *q;
-  u8                  temp_v[MAP_SIZE >> 3];
+  u32                 len = (afl->fsrv.map_size >> 3);
   u32                 i;
 
+  if (len == 0) len = 1;
+
   if (afl->dumb_mode || !afl->score_changed) return;
 
   afl->score_changed = 0;
 
-  memset(temp_v, 255, MAP_SIZE >> 3);
+  u8 temp_v[len];
```

Comment:
AFAIK most code styles prevent the use of variable length arrays.
They have a tendency to break software in random ways (as the stack may explode, etc.) and lead to a lot of slower added code. Also, they are C99+. In the case of the Linux kernel, seee https://lwn.net/Articles/749064/ 
Instead, I suggest (re)using a shared buffer in `afl` that you `ck_maybe_grow` to the `len` before use - or leave it at a size with fixed maximum.